examples/transmission_main_example.py

```diff
@@ -77,10 +61,63 @@ def main(args):
     axis('off')
     show()
 
+
+def main(args):
+    input_filename = str(args.filename)
+    split_filename = input_filename.split(".")
+    json_filename = split_filename[0]+'.json'
```

Comment:
`args.filename` is a `Path` object. Prefer use of native operations rather than coercion to string and use of string manipulation.

e.g., 
```
p = Path('a/b/c/d.txt')
p.with_suffix('.json')
```